truffle/src/main/java/org/jruby/truffle/nodes/core/ArrayNodes.java

```diff
@@ -1898,20 +1981,38 @@ public RubyArray mapInPlaceNull(RubyArray array, RubyProc block) {
         }
 
         @Specialization(guards = "isIntegerFixnum")
-        public RubyArray mapInPlaceFixnumInteger(VirtualFrame frame, RubyArray array, RubyProc block) {
+        public Object mapInPlaceFixnumInteger(VirtualFrame frame, RubyArray array, RubyProc block) {
+
```

Comment:
Please try to avoid these extra blank lines - we like them between basic blocks but not between the head of the method and the first bit of code. Sorry to be pedantic!